src/library_webgpu.js

```diff
@@ -1718,12 +1850,13 @@ var LibraryWebGPU = {
     return data;
   },
 
+  // In webgpu.h offset and size are passed in as size_t.
+  // And we assume that size_t is always 32bit in emscripten.
   wgpuBufferGetMappedRange: function(bufferId, offset, size) {
     var bufferWrapper = WebGPU.mgrBuffer.objects[bufferId];
     {{{ gpu.makeCheckDefined('bufferWrapper') }}}
 
-    // TODO: if the sentinel value becomes WGPU_WHOLE_SIZE instead of 0, update this.
-    if (size === 0) size = undefined;
```

Comment:
This breaking change could be difficult to debug, so maybe we should add a temporary
```js
if (size === 0) warnOnce('getMappedRange size=0 no longer means WGPU_WHOLE_MAP_SIZE');
```